core/templates/dev/head/domain/collection/CollectionRightsBackendApiService.js

```diff
@@ -20,24 +20,19 @@ oppia.factory('CollectionRightsBackendApiService', [
     '$http', '$q', 'COLLECTION_RIGHTS_URL_TEMPLATE', 'UrlInterpolationService',
     function($http, $q, COLLECTION_RIGHTS_URL_TEMPLATE,
       UrlInterpolationService) {
-      var _setCollectionStatus = function(
-          collectionId, collectionVersion, isPublic, successCallback,
-          errorCallback) {
+      var SetCollectionRights = function(
+        collectionId, putParams, successCallback, errorCallback) {
```

Comment:
Prefer passing in all args explicitly (like they were before), rather than in an opaque putParams blob. Might be simpler to do this if we don't have an is_publicized option.

If you prefer, it's also fine to expose a separate URL endpoint for the task of adding new authors to a collection, so that you don't have to pass different params here based on a 'mode' (which is a bit of an antipattern).